### PR TITLE
Expand the stalk command. Closes #137

### LINKED ISSUE #137: ;stalk feature request by numb
image

+contest

### PR DESCRIPTION
**Pull request by krofna**
- sorting by difficulty or time
- filter by contest or practice

### CODE REVIEW

File: tle/cogs/codeforces.py [hunk 1]
```diff
@@ -131,22 +131,30 @@ def __init__(self, bot):
...
+        type = 'CONTESTANT' if contest else 'PRACTICE'
+        handles = [arg for arg in args if arg[0] != '+']
+        handles = handles or ('!' + str(ctx.author),)
+        handles = await cf_common.resolve_handles(ctx, self.converter, handles)
+        submissions = [await cf.user.status(handle=handle) for handle in handles]
+        submissions = list({sub.problem.name : sub for subs in submissions for sub in subs
+                            if sub.verdict == 'OK' and sub.author.participantType == type}.values())
+
+        if hardest:
+            submissions.sort(key=lambda sub: sub.problem.rating if sub.problem.rating else 0, reverse=True)
```
Comment (meooow25) [inline tle/cogs/codeforces.py]: `lambda sub: sub.problem.rating or 0`

File: tle/cogs/codeforces.py [hunk 2]
```diff
@@ -131,22 +131,30 @@ def __init__(self, bot):
...
+        else:
+            submissions.sort(key=lambda sub: sub.creationTimeSeconds, reverse=True)
 
+        problems = [sub.problem for sub in submissions]
         msg = '\n'.join(f'[{prob.name}]({prob.url}) [{prob.rating if prob.rating else "?"}]'
                         for prob in problems[:5])
```
Comment (meooow25) [inline tle/cogs/codeforces.py]: We should paginate and display all.

Comment (krofna) [reply tle/cogs/codeforces.py]: Should we though? There could be thousands.

Comment (meooow25) [reply tle/cogs/codeforces.py]: Uh... good point 😬 
In that case, we can still increase it to ~10 and it should fit without pagination.

**Pull request merged**